onnxruntime/core/session/inference_session.cc

```diff
@@ -508,6 +508,19 @@ common::Status InferenceSession::RegisterExecutionProvider(const std::shared_ptr
     }
   }
 
+
+#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_MINIMAL_BUILD_CUSTOM_OPS)
+  // Create Custom Op if EP requests it
+  std::vector<OrtCustomOpDomain*> custom_op_domains;
+  p_exec_provider->GetCustomOpDomainList(custom_op_domains);
+
+  if (custom_op_domains.size() > 0) {
```

Comment:
nit: using empty() expresses the intent better.
```suggestion
  if (!custom_op_domains.empty()) {
```